app/controllers/chef_ingredients_controller.rb

```diff
@@ -0,0 +1,8 @@
+class ChefIngredientsController < ApplicationController
```

Comment:
Nice controller organization! Using a joins controller here is a great way to handle the need for ingredients for a given chef. 

Another option here could be to nest the ingredients controller under a folder called `chefs` so that the class definition would be `class Chefs::IngredientsController`.

Both of these implementations are good! It is a matter of preference!